src/add-to-app/android/project-setup.md

```diff
@@ -293,6 +293,44 @@ dependencies {
   releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
 }
 ```
+</br>
+
+##### Kotlin DSL
+
+<?code-excerpt title="MyApp/app/build.gradle.kts"?>
+```gradle
+android {
+    buildTypes {
+        release {
+      ...
+        }
+        debug {
+          ...
+        }
+
+        create("profile") {
+            initWith(getByName("debug"))
+        }
+}
+
+dependencies {
+  // ...
+  debugImplementation 'com.example.flutter_module:flutter_debug:1.0'
+  releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
```

Comment:
Trying this out with a freshly created Android app + Flutter module and it doesn't seem to work. Is this correct syntax?

If you haven't already, can you validate these changes with a similar setup and comment the steps you took? That's how most developers would land on this page. It would also help me review this because I'm not an add-to-app expert.